lib/tabula.js

```diff
@@ -126,10 +126,8 @@ function tabulaFormat(items, options) {
     assert.optionalArrayOfString(options.sort, 'options.sort');
     assert.optionalArrayOfString(options.validFields, 'options.validFields');
     assert.optionalBool(options.dottedLookup, 'options.dottedLookup');
+    assert.ok(options.columns || items[0], 'column names cannot be determined');
 
-    if (items.length === 0) {
-        return '';
-    }
     var cols = [];
```

Comment:
``` js
if (!options.columns && items.length === 0) {
  return '';
};
```

this way, it'll return nothing if the columns can't be determined either implicitly or explicitly